src/main/resources/templates/fds/subNavigation.ftl

```diff
@@ -24,7 +24,8 @@
 </#macro>
 
 <#macro subNavigationSectionItem linkAction linkName currentItemHref>
-  <#if currentItemHref?starts_with(linkAction)>
```

Comment:
Changes here were to fix the styling on the sub nav for the current endpoint. Tags have spaces where as the request comes in with the %20 URL encoding for spaces.